tools/depends/target/libspdlog/Makefile

```diff
@@ -28,7 +28,7 @@ else
     RETRIEVE_TOOL_FLAGS := -Ls --create-dirs -f -O
     ARCHIVE_TOOL := tar
     ARCHIVE_TOOL_FLAGS := --strip-components=1 -xf
-    CMAKE := cmake -DCMAKE_INSTALL_PREFIX=$(PREFIX)
+    CMAKE := cmake -DCMAKE_INSTALL_PREFIX=$(PREFIX) -DCMAKE_PREFIX_PATH=${PREFIX}
```

Comment:
Does `${PREFIX}` work the same as `$(PREFIX)` and is there any reason for not using the same style for both?

Reply:
yes, it's the same. We should still try to use the same style

Reply:
So the preferred style is `$(PREFIX)`? So using round brackets. Good to know.

Reply:
Sorry too much `cmake` syntax recently ;-) Fixed.